# data output from RANSAC feature matching and respective point clouds
geometry_msgs/Transform featureTransform
sensor_msgs/PointCloud2 sourcePointcloud
sensor_msgs/PointCloud2 targetPointcloud
match[] matches
